lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -86,7 +88,10 @@ $-separator-color: var(--nx-swatch-indigo-40);
 }
 
 .nx-global-sidebar__logo {
-  height: $-logo-height;
+  // a temporary stop gap to ensure no logos break the header container - matches plaid-villain logo width
+  max-width: 155px;
```

Comment:
I guess we should really have another logo and visual test to check this case

Reply:
> I guess we should really have another logo and visual test to check this case

The regular snapshots do mostly this already, since the Plaid Villain logo breaks the container without a max-width in place. However, I did add another snapshot with the different sized logo here: [39c4f95](https://github.com/sonatype/sonatype-react-shared-components/pull/1078/commits/39c4f9579385ab12e53f3c20e1a4b067d13ccb5e)

Reply:
> The regular snapshots do mostly this already, since the Plaid Villain logo breaks the container without a max-width in place

Good point